src/main/java/gov/usgs/wma/waterdata/parameter/Domain.java

```diff
@@ -0,0 +1,12 @@
+package gov.usgs.wma.waterdata.parameter;
+
+import java.util.List;
+
+public enum Domain {
+	//any,
+	groundwater_levels;
+
+	public static boolean includesGroundWaterLevels(List<Domain> domains) {
+		return domains != null && domains.contains(groundwater_levels);
+	}
+}

```

Comment:
Is this only used for the data endpoint?  It seems like domain applies to all endpoints.

Reply:
The idea is that this will soon have other values like Instantaneous, etc. Yes, it potentially could be used by the
other end points.